client/my-sites/patterns/components/get-started/style.scss

```diff
@@ -0,0 +1,72 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.patterns-get-started--body {
+	.patterns-get-started--body-buttons {
+		padding: 0 110px;
+		margin-top: -32px;
+		margin-bottom: 48px;
+
+		.patterns-get-started--body-buttons-build {
+			font-family: $font-sf-pro-text;
+			font-size: rem(13px);
+			color: var(--studio-white);
+			height: 40px;
+			padding: 0 17px;
+			border-radius: 2px;
+			background: #3858e9;
+		}
+	}
+
+	.patterns-get-started--body-list {
+		overflow: scroll;
+
+		.patterns-get-started--body-list-inner {
+			min-width: 1220px;
+			padding: 0 110px;
+			display: flex;
+			gap: 24px;
+
+			.patterns-get-started--body-list-item {
+				flex: 1;
+				display: block;
+				height: auto;
+				padding: 0;
+
+				.patterns-get-started--body-list-item-image {
+					width: 100%;
+					padding-bottom: 16px;
+				}
+
+				.patterns-get-started--body-list-item-name {
```

Comment:
Just thoughts out loud - I am not a fan of BEM, but I am big fan of nesting, it was the biggest game-changer in CSS, since I am using it (about 15 years 😅), and I am still in love with nesting 🙂 Main reason - it helps to keep CSS structured, encapsulated and easy to read at a glance (mostly if you see the specific code the first time, so it's easy to understand it and not to break anything).

>I guess we could also theoretically opt not to use BEM, in which case we could do the full nesting. I don't think we have any official guidelines or requirements to use BEM. It's more of a convention.

Honestly, I am totally ok with using BEM or not using, but I would be very happy to use nesting in our codebase.
So, according to your opinion, looks like you are ok with - not using BEM and using nesting? If so, I can rewrite all classNames in this PR.

Reply:
> I guess we could also theoretically opt not to use BEM

It looks like we do in fact have an ESLint rule that's meant to enforce BEM class names ([rule config](https://github.com/Automattic/wp-calypso/blob/nightnei/patternsHomePage/.eslintrc.js#L425), docs: https://wpcalypso.wordpress.com/devdocs/packages/eslint-plugin-wpcalypso/docs/rules/jsx-classname-namespace.md).

There might be an issue with that plugin, because I don't see ESLint emitting those errors, but we should stick to that intention and keep using BEM.

Reply:
Adjusted all components